### PR TITLE
GUI-2273: Enable sortable columns for volumes table on landing page

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2273

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1203/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1206/
Test PASSed.

File: eucaconsole/static/css/pages/volumes.css [hunk 1]
```diff
@@ -9,6 +9,7 @@
 ----------------------------------------- */
 /* @section Landing pages styles */
 .columns.datagrid { padding-left: 0; padding-right: 1rem; padding-bottom: 2rem; }
+.columns.datagrid.has-smart-table.tableview #sorting-controls { display: none; }
```
Comment (dkavanagh) [inline eucaconsole/static/css/pages/volumes.css]: Shouldn't the sort be removed from the landing page? Is this a stop-gap till every page has smart-table?

Comment (kamalgill) [inline eucaconsole/static/css/pages/volumes.css]: @dkavanagh: The tile view still needs the sorting drop-down

**Pull request merged**